webapp/src/main/resources/config/application.properties

```diff
@@ -206,6 +206,9 @@ spring.session.jdbc.table-name=SPRING_SESSION_V2
 # Notification for branch activity
 #l10n.branchNotification.slack.enabled=true
 #l10n.branchNotification.slack.userEmailPattern={0}@test.com
+# Blocked usernames to not send Slack messages to (Regex works here and usernames can be seperated by commas)
+#l10n.branchNotification.slack.blockedUsernames=bot-number-001,jenkins-bot,.*\\\[bot\\\]$
```

Comment:
We should throw an error on startup if an invalid regex is configured

Reply:
Done!